### DISCUSSION TITLE
Second job causing errors / manager not starting

### DISCUSSION THREAD

**Discussion by s3g4-noll** [swar/Swar-Chia-Plot-Manager#1164] [Q&A]
Hi, 

I'm pretty new to this, but I want to create different jobs to maximize my plotting. 

The first job on its own is just working fine, but the second one is causing errors.

![plot1](https://user-images.githubusercontent.com/61414953/121143079-f04b6d00-c83c-11eb-9e79-d8126bb3ecef.PNG)

This is what happens, when the second job is commented out:
![plot2](https://user-images.githubusercontent.com/61414953/121143169-0c4f0e80-c83d-11eb-8bf8-d932c24546ad.PNG)

The error messages don't help much and I don't know what went wrong here.

**Comment by MrT63** [comment]
It says where there error is- Line 7... basically there should not be an empty space before the line and the location of your chia directpory shud be accurate:  Line 7 is the chia location line....replace "Tristan" with your user name , and also the app versionj number

# This is a single variable that should contain the location of your chia executable file. This is the blockchain executable.
#
# WINDOWS EXAMPLE: C:\Users\Tristan\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia.exe
#   LINUX EXAMPLE: /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/chia
#  LINUX2 EXAMPLE: /home/swar/chia-blockchain/venv/bin/chia

chia_location: C:\Users\Tristan\AppData\Local\chia-blockchain\app-1.1.6\resources\app.asar.unpacked\daemon\chia.exe
